input/includes/search_parameters_intro.md

```diff
@@ -1,3 +1,5 @@
-Below is an overview of the mandatory and optional search parameters and combined search parameters. FHIR search operations and the syntax used to describe the interactions is described <a href="http://hl7.org/fhir/R4/search.html">here</a>.
+Below is an overview of the mandatory and optional search parameters and combined search parameters. See the [AU Core CapabilityStatements](capability-statements.html) for a complete list of supported RESTful interactions for this IG.
+
+FHIR search operations are described <a href="http://hl7.org/fhir/R4/search.html">here</a> and the syntax used to describe the interactions is described in AU Core is defined <a href="general-guidance.html#readsearch-syntax">here</a>.
```

Comment:
yes, copy paste error, should be "FHIR search operations are described [here](http://hl7.org/fhir/R4/search.html) and the syntax used to describe AU Core interactions is defined [here](https://github.com/hl7au/au-fhir-core/pull/general-guidance.html#readsearch-syntax)."

kept defined so that it didn't have too many uses of describe